app/templates/components/frost-modal.hbs

```diff
@@ -0,0 +1,47 @@
+{{#ember-remodal
```

Comment:
Generic modal component using remodal. slots for 'target' component used to launch the modal, header, scrollable body, cancel button and confirm button. Do we need to support yielding of more buttons? Footer div style is always present.